scripts/build.ps1

```diff
@@ -165,101 +165,228 @@ function Invoke-TestAssetsBuild {
         Write-Log ".. .. Build: Source: $TPB_TestAssets_Solution -- add NuGet source"
         Invoke-Exe -IgnoreExitCode 1 $nugetExe -Arguments "sources add -Name ""locally-built-testplatform-packages"" -Source $env:TP_TESTARTIFACTS\packages\ -ConfigFile ""$nugetConfig"""
         Invoke-Exe $dotnetExe -Arguments "build $TPB_TestAssets_Solution --configuration $TPB_Configuration -v:minimal -p:CIBuild=$TPB_CIBuild -p:LocalizedBuild=$TPB_LocalizedBuild -bl:""$env:TP_OUT_DIR\log\$Configuration\TestAssets.binlog"""
+    }
+    finally {
+        Write-Log ".. .. Build: Source: $TPB_TestAssets_Solution -- remove NuGet source"
```

Comment:
I think we have a helper method for "intended log writing". Could you use it instead?

Reply:
Hum I cannot find it back. So nevermind this comment.

Reply:
Are you mistaking Write-Log for Write-Host? :)